Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:18.04
+FROM alpine:latest
 
 RUN apt-get -y update && \
```

Comment:
Alpine does no use aptitude package manager.

Reply:
Is there another image you'd recommend using instead?